### ISSUE TITLE
Track W3C Verifiable Claims work - does schema.org fit somewhere?

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#1654]
see https://www.w3.org/TR/verifiable-claims-use-cases/

**Comment by mediaprophet** [comment]
I think it does.

**Comment by mediaprophet** [comment]
https://github.com/w3c/vc-data-model/issues/33

**Comment by mrtoddalexander** [comment]
http://schema.org/ClaimReview

**Comment by msporny** [comment]
@danbri we are re-using schema.org where we can. It is a VCWG goal to ensure that its easy for search crawlers to eventually pick up on Verifiable Claims published as JSON-LD in web pages. This is a core use case. So, whatever we need to do to make that as easy as possible for you, let us know.

We have not yet decided on the URLs for the vocabularies, but if you think we could put the core vocabulary in schema.org (and rapidly iterate), then we should certainly make sure the VCWG is aware of that and works to that end.

In other words, we want to work w/ schema.org. The only question is how much interest there is from the schema.org side, how practical it will be to launch w/ ONLY schema.org, and what the vocabulary management implications are for doing so. Since we're dealing w/ digital signatures, legacy data could be corrupted if the wrong sorts of updates happen w/ schema.org. Happy to discuss it more in person at TPAC if you'd like... or even request a slot for you to speak to the VCWG at W3C TPAC (we meet Thu/Fri).

**Comment by rvguha** [comment]
Manu,

 Schema.org is super interested in making this happen.

guha

**Comment by msporny** [comment]
@rvguha Great, thanks for the response. I'll make sure the VCWG is aware of the desire to collaborate and we'll develop several strategies to get this stuff integrated w/ schema.org.

**Comment by danbri** [comment]
@manu - can you give us an updated on where the VF work has gotten to?

**Comment by msporny** [comment]
The WG is making solid progress, most of the difficult design decisions have been made and now we're wrapping up a number of minor items. 

The latest spec is here: https://w3c.github.io/vc-data-model/ 

We are expecting to go into Candidate Recommendation in October 2018. I think it would be a coup if we could add the vocabulary terms to schema.org. What would be the best first step for doing this? Summarizing the terms that needed to be added to schema.org (there are only a handful)?

**Comment by danbri** [comment]
@msporny - can you give us an update on where this work ended up?

**Comment by akuckartz** [comment]
@danbri I think that this answers the question:
https://github.com/w3c/vc-data-model/issues/194#issuecomment-420007644

**Comment by danbri** [comment]
@akuckartz it helps, and the spec reached REC at W3C late last year. They took the route of creating their own vocabulary and showing [a non-normative](https://www.w3.org/TR/vc-data-model/#example-41-a-credential-uniquely-identifying-a-subject) example of Schema.org being plugged in. But I'm still interested (@msporny and others) in any update on how the implementation landscape is looking. For example are there vocabularies used for Verifiable Claims which could be usefully contributed into Schema.org?
